src/__tests__/index.js

```diff
@@ -468,17 +459,13 @@ test('can accept functions which return string class names', () => {
 test('should accept user defined contextTypes', () => {
   const dynamicStyles = jest.fn()
   const Child = glamorous.div(dynamicStyles)
-  Child.contextTypes = {
-    fontSize: PropTypes.number,
-  }
+  Child.contextTypes = {fontSize: PropTypes.number}
 
-  const context = {
-    fontSize: 24,
-  }
+  const context = {fontSize: 24}
 
   render(<Child />, {context})
   expect(dynamicStyles).toHaveBeenCalledTimes(1)
   const theme = {}
-  const props = {}
+  const props = {theme}
```

Comment:
Here's a change. `theme` will now always be a part of the `props` on a component. I don't see that as a problem... If the user is using the `theme` prop, we have special behavior for that already (and that behavior remains the same).

In fact, in a future breaking change, I would like to remove the `theme` as the second argument and just make it part of the props, so instead of:

```javascript
glamorous.div((props, theme) => theme.someThing)
```

You could just do:

```javascript
glamorous.div(({theme}) => theme.someThing)
```